examples/stdlib/exception/combinators.effekt

```diff
@@ -0,0 +1,75 @@
+module examples/pos/exception/combinators
+
+import exception
+import result
+
+/**
+ * Used as a type for `Exception` purely for independent testing
+ */
+record TestException()
+
+/**
+ * Returns the string if index > 0; otherwise raises a TestException.
+ */
```

Comment:
Please use `///` instead: I'm planning to deprecate multiline comments soon. :)
```suggestion
/// Used as a type for `Exception` purely for independent testing
record TestException()

/// Returns the string if index > 0; otherwise raises a TestException.
```